android/local.properties

```diff
@@ -1,5 +1,5 @@
-sdk.dir=C:\\Users\\PATRYK\\AppData\\Local\\Android\\sdk
-flutter.sdk=F:\\Flutter\\flutter
+sdk.dir=C:\\Users\\Gnosia\\AppData\\Local\\Android\\sdk
+flutter.sdk=C:\\src\\Flutter\\flutter
```

Comment:
Remember to not post this file. I will try to add it to the gitignore one more time.